nesta/core/scripts/nesta_docker_build

```diff
@@ -73,7 +73,8 @@ cd ${TOP_DIR}
 DKR_IMG_NAME=${AWS_DKR_PATH}/${DKR_PKG_NAME}:latest
 docker tag ${DKR_PKG_NAME}:latest ${DKR_IMG_NAME}
 docker push ${DKR_IMG_NAME}
-docker system prune -a -f 
```

Comment:
this was a bit heavy-handed, causing all images (including those in production) to be deleted...